TW-286 proxy session docs (#3793)
postmanlabs/postman-docs: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md

---

> If you are using APIs to build client-side applications—mobile apps, websites, or desktop applications, you may want to see the actual HTTP and HTTPS request traffic that's being sent and received in the application. Sometimes you might discover APIs that aren't even documented. You can capture this network traffic using the proxy that's built into the Postman app.

Author: (Removed the dashes as part of another comment...)

When you select one or more cookies from the list on the **Cookies** tab, you can use the following controls:

* **+ Add to Cookie Jar** - add the request to the Postman cookie jar.

* <img alt="External link icon" src="https://assets.postman.com/postman-docs/icon-delete-v9.jpg" width="12px" style="vertical-align:middle;margin-bottom:5px"> - delete the cookie.

The bottom right corner shows the total time of the capture session, along with the total size of captured traffic. If you want to temporarily stop the capture session, select the **Pause** button, and select **Resume** to restart it.

### Step 5: Stop the debug session and proxy

When you're done with your debug session, select **Stop** in the lower right corner. This will finish the debug session, and results of the debug session will be saved in the **History** tab.

After stopping the session, the proxy is still running. Select the **Capture Requests** icon in the Postman footer, and you can either start another debug session, or select **Disable proxy** to turn off the proxy.

### Step 6: View the proxy debug session results

> After completing a debug session, a transaction of it is located in the **History** tab, named **Proxy debug session**.

Reviewer: "Transcription" here seems a little tricky. Maybe something like:

```suggestion
After completing a debug session, details from the session are located in the **History** tab, named **Proxy debug session**.
```

❓

When you open a saved debug session, the top header displays the session start time, total size, duration, and source.

> The **Overview** - displays summary graphs of the traffic captured in the session. You can select graphs summarizing traffic by method, domain, data mode, or return status code.

Reviewer: ```suggestion
The **Overview** displays summary graphs of the traffic captured in the session. You can select graphs summarizing traffic by method, domain, data mode, or return status code.
```

Author: (already fixed)
